.ci/build_kubernetes.sh

```diff
@@ -9,18 +9,25 @@ echo ${DOCKER_PASSWORD} | docker login --username ${DOCKER_USERNAME} --password-
 FIRST_DOMAIN_HOSTNAME=${JOB_NAME}.${DEVELOPMENT_SERVER_DOMAIN}
 SECOND_DOMAIN_HOSTNAME=2.${JOB_NAME}.${DEVELOPMENT_SERVER_DOMAIN}
 
-# Set domain name into ingress controller so ingress can listen on domain name
-yq write --inplace project-base/kubernetes/ingress.yml spec.rules[0].host ${FIRST_DOMAIN_HOSTNAME}
-yq write --inplace project-base/kubernetes/ingress.yml spec.rules[1].host ${SECOND_DOMAIN_HOSTNAME}
-
-# Set domain into dev-stack (redis-admin, elasticsearch, adminer) hostnames
-yq write --inplace project-base/kubernetes/kustomize/overlays/ci/ingress-patch.yaml [0].value.host adminer.${FIRST_DOMAIN_HOSTNAME}
-yq write --inplace project-base/kubernetes/kustomize/overlays/ci/ingress-patch.yaml [1].value.host elasticsearch.${FIRST_DOMAIN_HOSTNAME}
-yq write --inplace project-base/kubernetes/kustomize/overlays/ci/ingress-patch.yaml [2].value.host redis-admin.${FIRST_DOMAIN_HOSTNAME}
+FILES=(
+    project-base/kubernetes/ingress.yml
+    project-base/kubernetes/kustomize/overlays/ci/ingress-patch.yaml
+    project-base/kubernetes/deployments/webserver-php-fpm.yml
+)
```

Comment:
What about to get all files recursively from path `kubernetes`? This can easily make a mistake, because you have to think that you need to add file path here.

Reply:
That would be really nice to see in it. I will train my bash skills for next level ;)